api/notd/gallery_manager.py

```diff
@@ -161,7 +161,8 @@ async def query_collection_tokens(self, registryAddress: str, limit: int, offset
         usesListings = isListed or minPrice or maxPrice
         if collection.doesSupportErc721:
             query = (
-                sqlalchemy.select(TokenMetadatasTable, TokenCustomizationsTable, TokenBestListingsView, sqlalchemy.literal(1).label('quantity'))
+                sqlalchemy.select(TokenMetadatasTable, TokenCustomizationsTable, TokenOwnershipsTable, TokenBestListingsView, sqlalchemy.literal(1).label('quantity'))
```

Comment:
i dont think we should need the ownership in the select, the join might be enough?